client/app/components/Input/InputTextarea.scss

```diff
@@ -57,11 +57,12 @@
     }
 
     &Selected {
-      color: $key-lime;
+      background: white;
```

Comment:
This totally sufficient, but for consistency's sake, let's use [$white](https://github.com/ifmeorg/ifme/blob/main/app/assets/stylesheets/base/_colors.scss#L18)!

Reply:
Great, just changed it!